glass/src/libnt/native/cpp/NTField2D.cpp

```diff
@@ -155,18 +134,21 @@ NTField2DModel::NTField2DModel(NT_Inst inst, std::string_view path)
     : m_nt{inst},
       m_path{fmt::format("{}/", path)},
       m_name{m_nt.GetEntry(fmt::format("{}/.name", path))} {
+#if 0
```

Comment:
Delete these?

Reply:
Whoops, these are actually a TODO to update.